src/fabric/src/fabric_doc_update.erl

```diff
@@ -106,11 +106,13 @@ before_doc_update(DbName, Docs, Opts) ->
         {true, _} ->
             %% fake db is expensive to create so we only do it if we have to
             Db = fabric_util:fake_db(DbName, Opts),
-            [couch_replicator_docs:before_doc_update(Doc, Db) || Doc <- Docs];
+            [couch_replicator_docs:before_doc_update(Doc, Db, interactive_updates)
+                || Doc <- Docs];
         {_, true} ->
             %% fake db is expensive to create so we only do it if we have to
             Db = fabric_util:fake_db(DbName, Opts),
-            [couch_users_db:before_doc_update(Doc, Db) || Doc <- Docs];
+            [couch_users_db:before_doc_update(Doc, Db, interactive_updates)
```

Comment:
I think it should be `interactive_edit` unless `interactive_updates` is a new thing defined in some other dependent PR I am not aware of.

Reply:
yes, change it to interactive_edit